kernels/python-maths/requirements.in

```diff
@@ -1,28 +1,28 @@
 # math-base
-h5py==3.6.0
-matplotlib==3.5.1
-numpy==1.22.3
-plotly==5.7.0
-scikit-learn==1.0.2
-scipy==1.8.0
-sympy==1.10.1
+h5py
+matplotlib
+numpy
+plotly
+scikit-learn
+scipy
+sympy
 
-pandas==1.4.2
+pandas
 # optional extensions for pandas
-openpyxl==3.0.9
-xlrd==2.0.1
-xlwt==1.3.0
+openpyxl
+xlrd
+xlwt
 
 # jupyter-extensions
-ipyevents==2.0.1
-ipympl==0.8.8
-ipywidgets==7.7.0
-jupyterlab==3.4.4
-jupyterlab_latex==3.1.0
-jupyterlab_widgets==1.1.0
-jupyterlab-git==0.36.0
-jupyterlab-lsp==3.10.1
-octave_kernel==0.34.2
-python-lsp-server[all]==1.4.1
-voila==0.3.5
-watchdog[watchmedo]==2.1.7
\ No newline at end of file
+ipyevents
+ipympl
+ipywidgets
+jupyterlab
```

Comment:
I also like: `jupyterlab-code-formatter`, `black`, `pylsp-mypy`, `isort` but not sure if its too much

Reply:
Do users use those? Will add them if they're used a lot. Although we don't have a way to keep track of what users use...